lib/facter/env_temp_variable.rb

```diff
@@ -0,0 +1,3 @@
+Facter.add(:env_temp_variable) do
+  setcode { ENV['TEMP'] }
```

Comment:
Do we want to .gsub what is returned by the ENV variable? With the same regards then could we trust the system32 fact?